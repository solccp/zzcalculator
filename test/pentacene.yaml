---
structure_id: 1
number_of_atoms: 22
double_bonds:
- [2, 3]
- [10, 9]
- [11, 12]
- [18, 17]
- [16, 15]
- [13, 14]
- [8, 7]
- [4, 5]
- [1, 20]
- [6, 19]
- [21, 22]
---
structure_id: 2
number_of_atoms: 22
double_bonds:
- [2, 3]
- [10, 9]
- [11, 12]
- [18, 17]
- [16, 15]
- [13, 14]
- [8, 7]
- [4, 5]
- [1, 6]
- [19, 21]
- [20, 22]
---
structure_id: 3
number_of_atoms: 22
double_bonds:
- [2, 3]
- [10, 9]
- [11, 12]
- [18, 17]
- [16, 15]
- [13, 14]
- [8, 7]
- [4, 5]
rings:
- [1, 6, 19, 21, 22, 20]
---
structure_id: 4
number_of_atoms: 22
double_bonds:
- [1, 2]
- [20, 22]
- [21, 19]
- [6, 5]
- [3, 10]
- [4, 7]
- [8, 14]
- [9, 11]
- [12, 18]
- [13, 15]
- [16, 17]
---
structure_id: 5
number_of_atoms: 22
double_bonds:
- [1, 2]
- [20, 22]
- [21, 19]
- [6, 5]
- [3, 10]
- [4, 7]
- [8, 14]
- [9, 11]
- [12, 13]
- [15, 16]
- [17, 18]
---
structure_id: 6
number_of_atoms: 22
double_bonds:
- [1, 2]
- [20, 22]
- [21, 19]
- [6, 5]
- [3, 10]
- [4, 7]
- [8, 14]
- [9, 11]
rings:
- [12, 13, 15, 16, 17, 18]
---
structure_id: 7
number_of_atoms: 22
double_bonds:
- [1, 2]
- [20, 22]
- [21, 19]
- [6, 5]
- [3, 10]
- [4, 7]
- [8, 9]
- [11, 12]
- [14, 13]
- [15, 16]
- [17, 18]
---
structure_id: 8
number_of_atoms: 22
double_bonds:
- [1, 2]
- [20, 22]
- [21, 19]
- [6, 5]
- [3, 10]
- [4, 7]
- [15, 16]
- [17, 18]
rings:
- [8, 9, 11, 12, 13, 14]
---
structure_id: 9
number_of_atoms: 22
double_bonds:
- [1, 2]
- [20, 22]
- [21, 19]
- [6, 5]
- [3, 4]
- [7, 8]
- [10, 9]
- [11, 12]
- [14, 13]
- [15, 16]
- [17, 18]
---
structure_id: 10
number_of_atoms: 22
double_bonds:
- [1, 2]
- [20, 22]
- [21, 19]
- [6, 5]
- [11, 12]
- [14, 13]
- [15, 16]
- [17, 18]
rings:
- [3, 4, 7, 8, 9, 10]
---
structure_id: 11
number_of_atoms: 22
double_bonds:
- [7, 8]
- [10, 9]
- [11, 12]
- [14, 13]
- [15, 16]
- [17, 18]
- [19, 21]
- [20, 22]
rings:
- [1, 2, 3, 4, 5, 6]
