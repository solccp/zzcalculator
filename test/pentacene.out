 6 + 5 x
 total: 11
